config/clusters/catalystproject-latam/common.values.yaml

```diff
@@ -0,0 +1,177 @@
+nfs:
+  enabled: true
+  pv:
+    mountOptions:
+      - soft
+      - noatime
+    # Google FileStore IP
+    serverIP: 172.21.82.226
+    # Name of Google Filestore share
+    baseShareName: /homes/
+jupyterhub:
+  custom:
+    2i2c:
+      add_staff_user_ids_to_admin_users: true
+      add_staff_user_ids_of_type: "github"
+    homepage:
+      templateVars:
+        org:
+          name: Catalyst Project
+          url: ""
+          logo_url: https://github.com/2i2c-org/infrastructure/assets/6181563/0667c576-7dce-4443-afd2-922288530814
+        designed_by:
+          name: "2i2c"
+          url: https://2i2c.org
+        operated_by:
+          name: "2i2c"
+          url: https://2i2c.org
+        funded_by:
+          name: Catalyst Project
+          url: ""
```

Comment:
@jmunroe there is `url` missing in `org` and `funded_by`. Do you have one to fill in?